seaworthy/ps.py

```diff
@@ -4,18 +4,27 @@
 
 
 class PsException(Exception):
-    pass
+    """
+    Exception indicating problems operating on process lists and trees.
+    """
 
 
 @attr.s
 class PsRow(object):
+    """
+    Representation of a process list entry.
```

Comment:
Nitpick: this is technically accurate but possibly not that easy to understand. Could maybe just add a sentence like `Contains details about a single running process.`